bbb/services.py

```diff
@@ -157,56 +157,56 @@ def _handleFinishedRequest(self, brid, properties, results):
             taskid = task.taskId
             runid = int(task.runId)
         except TaskNotFound:
-            log.debug("Task not found for brid %s, nothing to do.", brid)
+            log.debug("buildrequest %s: task not found; nothing to do.", brid)
             return
 
-        log.info("Handling finished task %s run %s brid %s", taskid, runid,
-                 brid)
+        log.info("buildrequest %s: task %s: run %s: handling finished event", brid, taskid, runid)
         # Try to claim the task in case if the "started" event comes after
         try:
             self.tc_queue.claimTask(taskid, runid, {
                 "workerGroup": self.tc_worker_group,
                 "workerId": self.tc_worker_id,
             })
-            log.info("Task %s run %s claimed unexpectedly", taskid, runid)
+            log.info("buildrequest %s: task %s: run %s: task/run claimed unexpectedly", brid, taskid, runid)
         except TaskclusterRestFailure:
-            log.debug("Cannot claim task %s run %s, assuming it is claimed already", taskid, runid)
+            log.debug("buildrequest %s: task %s: run %s: cannot claim task; assuming it is claimed already", brid, taskid, runid)
 
         # Attach properties as artifacts
-        log.info("Attaching properties to task %s", taskid)
+        log.info("buildrequest %s: task %s: attaching properties", brid, taskid)
         try:
             # Our artifact must expire at or before the task's expiration
             expires = self.tc_queue.task(taskid)['expires']
             createJsonArtifact(self.tc_queue, taskid, runid, "public/properties.json", properties, expires)
         except TaskclusterRestFailure as e:
-            log.exception("Caught exception when creating an artifact for %s (Task is probably already completed), not retrying...", taskid)
+            log.exception("buildrequest %s: task %s: caught exception when creating an artifact (Task is probably already completed), not retrying...",
+                          brid, taskid)
```

Comment:
listing runID is possibly useful on this error message, since its used in the createJsonArtifact above